cli/bin/semgrep

```diff
@@ -55,15 +55,36 @@ class CoreNotFound(Exception):
  
     def __str__(self):
         return(self.value)
-           
+
+# Similar to cli/src/semgrep/engine.py check_is_correct_pro_version
+def is_correct_pro_version(core_path):
+    # We want to be careful about what we import here in order to keep this
+    # script lightweight. However, this file just defines a single constant and
+    # it takes well under a millisecond to import this.
+    from semgrep import __VERSION__
+    # Duplicate of cli/src/semgrep/semgrep_core.py pro_version_stamp_path
+    stamp_path = core_path.parent / "pro-installed-by.txt"
```

Comment:
Yeah, since I'm having trouble getting the CI job running where we do just this, I'm now planning to add an option to `semgrep install-semgrep-pro` which will allow the caller to supply a binary instead of downloading it. that will hopefully help with the docker container too, since we can just call that instead of adding the stamp manually.

Reply:
@aryx So, I got all the CI jobs passing, and I think this should work fine with the docker container too. It will put the stamp alongside the `semgrep-core-proprietary` binary, so if it has permission to install that binary, it should have permission to write the stamp as well. Is it optimistic to assume that this is tested in CI? All the tests are green. If it's not, and this breaks something, I'll sort it out tomorrow.